.github/workflows/build_and_test.yml

```diff
@@ -107,6 +107,30 @@ jobs:
     - name: Clippy
       run: cargo clippy --all -- -D warnings
 
+  rust_ios:
+    name: Build Rust for iOS
+
+    runs-on: macos-latest
+
+    needs: changes
+
+    if: ${{ needs.changes.outputs.rust == 'true' }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        components: rustfmt, clippy
```

Comment:
Nitpicks: this custom action also has `target: aarch64-apple-ios` as a config option (if you only need one target).

(Also, we're not using Clippy in this build.)